python/lsst/cell_coadds/_single_cell_coadd.py

```diff
@@ -23,7 +23,8 @@
 
 __all__ = ("CoaddInputs", "SingleCellCoadd",)
 
-from collections.abc import Iterable, Set
+from collections import OrderedDict
```

Comment:
Regular Python `dict` has been ordered since Python 3.7; there's no need to use `OrderedDict` anymore.